docs/api/api/api.txt

```diff
@@ -1995,8 +1996,12 @@ POST /staging/<staging_workflow_project>/staging_projects/<staging_project>/stag
 XmlBody: remove_staged_requests
 XmlResult: status_ok
 
+DELETE /staging/<staging_workflow_project>/staging_projects/staged_requests/<request>
+  Remove a bs_request from the staged requests of a staging project.
+XmlResult: status_ok
+
 DELETE /staging/<staging_workflow_project>/staging_projects/<staging_project>/staged_requests
```

Comment:
Why do we need both routes? Why not only one route, without `StagingProject`, where I send XML with one or more requests in it?

Reply:
@hennevogel, I applied the changes that you suggest.